src/webui/api/torrentscontroller.cpp

```diff
@@ -217,6 +217,28 @@ namespace
             infoHashes << hash;
         return infoHashes;
     }
+
+    bool operator<(const QJsonValue& left, const QJsonValue& right)
+    {
+        Q_ASSERT(left.type() == right.type());
+
+        switch (left.type())
+        {
+        case QJsonValue::Bool:
+            return left.toBool() < right.toBool();
+        case QJsonValue::Double:
+            return left.toDouble() < right.toDouble();
+        case QJsonValue::String:
+            return left.toString() < right.toString();
+        case QJsonValue::Null:
+        case QJsonValue::Undefined:
+            return false;
+        default:
```

Comment:
This branch confuses. JSON does not have (and isn't supposed to have) other types than those listed above.